application-mocca-calendar-ui/src/main/resources/MoccaCalendar/Code/UpcomingEventsPanel.xml

```diff
@@ -0,0 +1,325 @@
+<?xml version="1.1" encoding="UTF-8"?>
+
+<!--
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+-->
+
+<xwikidoc version="1.3" reference="MoccaCalendar.Code.UpcomingEventsPanel" locale="">
+  <web>MoccaCalendar.Code</web>
+  <name>UpcomingEventsPanel</name>
+  <language/>
+  <defaultLanguage/>
+  <translation>0</translation>
+  <creator>xwiki:XWiki.Admin</creator>
+  <parent>WebHome</parent>
+  <author>xwiki:XWiki.Admin</author>
+  <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
+  <version>1.1</version>
+  <title>UpcomingEventsPanel</title>
+  <comment/>
+  <minorEdit>false</minorEdit>
+  <syntaxId>xwiki/2.1</syntaxId>
+  <hidden>false</hidden>
```

Comment:
Should be hidden no? Make sure you run ``xar:format``.

Reply:
Right, thanks. I had not realized that `xar:format` marks technical pages as hidden automatically only since 12.3RC1 from what I can see from the [documentation](https://dev.xwiki.org/xwiki/bin/view/Community/XARPlugin#HFormat26VerifyMojo). I marked the page as hidden manually in the latest commits.